test/integration/alias.pkg.js

```diff
@@ -1,17 +1,24 @@
 'use strict';
 
-const { test, beforeEach, afterEach } = require('tap');
+// const { test, beforeEach, afterEach } = require('tap');
 const FormData = require('form-data');
 const fetch = require('node-fetch');
 const path = require('path');
+const tap = require('tap');
 const fs = require('fs');
 
 const Server = require('../../services/fastify');
 const Sink = require('../../lib/sinks/test');
 
 const FIXTURE_PKG = path.resolve(__dirname, '../../fixtures/archive.tgz');
 
-beforeEach(async (done, t) => {
+// Ignore the timestamp for "created" field in the snapshots
+tap.cleanSnapshot = (s) => {
+    const regex = /"created": [0-9]+,/gi;
+    return s.replace(regex, '"created": -1,');
+};
```

Comment:
Did not know about cleanSnapshot

Reply:
Pretty neat feature. Makes it possible to have "random" data in the snapshots.